build.gradle

```diff
@@ -1,5 +1,6 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
+    ext.kotlin_version = '1.2.21'
     repositories {
         jcenter()
```

Comment:
add the mavenCentral() here (under jcenter()) instead of below in the other file.